sdks/go/pkg/beam/runners/dataflow/dataflow_test.go

```diff
@@ -197,3 +197,50 @@ func TestGetJobOptions_DockerNoImage(t *testing.T) {
 		t.Fatalf("getContainerImage() = %q, want %q", got, want)
 	}
 }
+
+func TestGetJobOptions_MappingNoUpdate(t *testing.T) {
```

Comment:
Could you update (here and similarly in the next test) to `TestGetJobOptions_TransformMappingNoUpdate` - if I didn't have the context of this pr I wouldn't know what was being tested.

Reply:
Also, its probably worth adding one more test for the happy path

Reply:
Done.